### ISSUE TITLE
Implement OrdNoBound and PartialOrdNoBound for structs

### ISSUE THREAD

**Issue by ggwpez** [paritytech/polkadot-sdk#2198]
Import, rebase and address comments please https://github.com/paritytech/substrate/pull/14635

(reserved for @adelarja - if he wants)

**Comment by adelarja** [comment]
@ggwpez Sure, I can work on this :muscle:

**Comment by adelarja** [comment]
@ggwpez

One question to clarify: I'm using [this](https://github.com/tmpolaczyk/substrate/tree/ord-no-bounds) repo to perform the import.

Here is what I did,

First, I added a new remote using the fork containing the branch with the code changes from the substrate repo:

```bash
git remote add source git@github.com:tmpolaczyk/substrate.git
```
Then I created a branch in my fork and ran:

```bash
git rebase source/ord-no-bounds
```

Afterward, I started fixing the conflicts and continued with the rebase. Is this the correct approach? Is there any way to rebase without having to address all these conflicts?
